metricflow-semantics/metricflow_semantics/specs/dunder_column_association_resolver.py

```diff
@@ -59,23 +59,13 @@ def visit_measure_spec(self, measure_spec: MeasureSpec) -> ColumnAssociation:  #
 
     def visit_dimension_spec(self, dimension_spec: DimensionSpec) -> ColumnAssociation:  # noqa: D102
         return ColumnAssociation(
-            column_name=StructuredLinkableSpecName(
-                entity_link_names=tuple(x.element_name for x in dimension_spec.entity_links),
-                element_name=dimension_spec.element_name,
-            ).qualified_name,
+            column_name=dimension_spec.qualified_name,
```

Comment:
If I had just ripped this whole class out when I first noticed it none of this would've happened. Lesson learned. Thanks for making the property access nature of this class even more obvious. :)